scripts/generate-component-config/build-component-config.js

```diff
@@ -3,6 +3,20 @@ const fs = require('fs');
 const kebabCase = require('lodash.kebabcase');
 const startCase = require('lodash.startcase');
 
+const blackListedDirectories = [
```

Comment:
blacklist and terraNameSpacedRepos seem like good candidates to pull out into a separate config. js, json, yml something like that.